client/src/app/(modules)/network/(form)/new/initiative/page.tsx

```diff
@@ -118,27 +107,28 @@ export default function ProjectForm() {
         .optional(),
       type: 'email',
     },
-    second_project_coordinator_name: {
-      label: 'Name',
+    project_coordinator_website: {
+      label: 'Website',
       zod: z
         .string()
         .max(255, {
-          message: 'Name is limited to 255 characters.',
+          message: 'Website is limited to 255 characters.',
         })
-        .optional(),
+        .regex(new RegExp('^(https?:\\/\\/)?(www\\.)?[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)+\\/?$'), {
+          message: 'Please, enter a valid URL.',
+        }),
```

Comment:
Can we add `.optional()` after the regex? Right now is marked as optional but is required by the form